rfd/0046-database-access-config.md

```diff
@@ -64,14 +66,269 @@ $ clusterawsadm bootstrap iam print-policy --document AWSIAMManagedPolicyControl
 
 Teleport has a `teleport configure` command that generates a sample config file.
 
-Database agent bootstrap commands will reside under `teleport db configure`
-family of subcommands. This aligns with our strategy of placing commands related
-to a particular service under its own subcommand namespace.
+Database agent bootstrap commands will reside under teleport db configure family
+of subcommands. This aligns with our strategy of placing commands related to a
+particular service under its subcommand namespace. It will provide subcommands
+for generating configuration files and bootstrapping database agents’ necessary
+configurations, for example, setting up IAM permissions for agents to proxy
+AWS-hosted databases.
+
+### Create config subcommand
+
+Similar to `teleport configure` but focused on configurations for Database
+agents. This command will generate a configuration file. It can also provide
+some examples of configuration, for instance, self-hosted databases setup.
+
+There will be flags to enable features like Aurora/RDS auto-discovery and
+dynamic registration. Each sample will show the required options with some
+explanation on how to fill them and documentation links.
+
+It will also provide flags to configure a specific database (similar to the
+flags present on the `teleport db start` command). When executed with these
+flags, the samples will not be included, only the static configuration of a
+single database.
+
+```bash
+teleport db configure create [--auth-server] [--token=] [--version=] [--enable-rds-auto-discovery] [--enable-dynamic-registration] [--output=] [--ca-pin] [--name] [--protocol] [--uri]
+```
+
+| Flag                            | Description |
+| ------------------------------- | ----------- |
+| `--auth-server`                 | Address of the auth server. |
+| `--token`                       | Invitation token to register with an auth server |
+| `--version`                     | Teleport configuration version. |
```

Comment:
I think this flag isn't really needed for db agent configuration. We can add it later if needed, let's remove for now.